internal/search/zoekt/zoekt.go

```diff
@@ -59,21 +60,21 @@ func parseRe(pattern string, filenameOnly bool, contentOnly bool, queryIsCaseSen
 	}, nil
 }
 
-func getSpanContext(ctx context.Context) (shouldTrace bool, spanContext map[string]string) {
+func getSpanContext(ctx context.Context, slog log.Logger) (shouldTrace bool, spanContext map[string]string) {
 	if !ot.ShouldTrace(ctx) {
 		return false, nil
 	}
 
 	spanContext = make(map[string]string)
 	if err := ot.GetTracer(ctx).Inject(opentracing.SpanFromContext(ctx).Context(), opentracing.TextMap, opentracing.TextMapCarrier(spanContext)); err != nil {
-		log15.Warn("Error injecting span context into map: %s", err)
+		slog.Warn("Error injecting span context into map: %s", log.Error(err))
 		return true, nil
 	}
 	return true, spanContext
 }
 
-func SearchOpts(ctx context.Context, k int, fileMatchLimit int32, selector filter.SelectPath) zoekt.SearchOptions {
-	shouldTrace, spanContext := getSpanContext(ctx)
+func SearchOpts(log log.Logger, ctx context.Context, k int, fileMatchLimit int32, selector filter.SelectPath) zoekt.SearchOptions {
```

Comment:
```suggestion
func SearchOpts(logger log.Logger, ctx context.Context, k int, fileMatchLimit int32, selector filter.SelectPath) zoekt.SearchOptions {
```

(careful about conflicting with the package name!)